lambda/variables.tf

```diff
@@ -88,3 +90,19 @@ variable "policy_attachments" {
   default     = []
   description = "A list of policy arns to attach. These will need to be pre-existing policies"
 }
+
+variable "log_retention" {
+  default = 30
+}
+
+variable "sqs_queue_mapping_batch_size" {
+  default = 1
+}
+
+variable "sqs_queue_batching_window" {
+  default = 0
+}
+
+variable "sqs_queue_concurrency" {
+  default = 2
+}
```

Comment:
Can this be optional as it will limit scalability?

Reply:
I've set the default to be null and then put a `dynamic` block around the `scaling_config` which only sets it if the concurrency is something other than null.